src/dmd/cparse.d

```diff
@@ -3075,6 +3075,17 @@ final class CParser(AST) : Parser!AST
             nextToken();
         }
 
+        /* Extension: add optional base type after the identifier
```

Comment:
```suggestion
        /* Clang extension: add optional base type after the identifier
```

Reply:
I think this was proposed for c2x as well, not sure if it made it in though

Reply:
It's not even mentioned in the current working draft.

Reply:
done